src/AstTransforms.cpp

```diff
@@ -1326,6 +1341,24 @@ bool PolymorphicObjectsTransformer::transform(AstTranslationUnit& translationUni
                         }
                     }
                 }
+
+                if (auto* aggregator = dynamic_cast<AstAggregator*>(node.get())) {
+                    if (isOverloadedAggregator(aggregator->getOperator())) {
+                        auto* targetExpression = aggregator->getTargetExpression();
+
+                        if (isFloat(targetExpression)) {
+                            AggregateOp convertedOperator = convertOverloadedAggregator(
+                                    aggregator->getOperator(), TypeAttribute::Float);
+                            aggregator->setOperator(convertedOperator);
+                            changed = true;
+                        } else if (isUnsigned(targetExpression)) {
```

Comment:
Symbol support for `min`, `max`, is finished (or should be very shortly; #1364 ). Do we add overloads for `symbol` now or delegate to a future PR?